bot/exts/evergreen/latex.py

```diff
@@ -71,6 +77,7 @@ def _prepare_input(text: str) -> str:
 
     @commands.command()
     @commands.max_concurrency(1, commands.BucketType.guild, wait=True)
+    @commands.cooldown(1, 5, commands.BucketType.guild)
```

Comment:
I'm not sure this is needed, since the issue isn't with too many people using it quickly, but 1 bad actor giving it really long input. I don't think this cooldown is needed, but I'm not completely against it if someone else disagrees.

Reply:
> 
> 
> I'm not sure this is needed, since the issue isn't with too many people using it quickly, but 1 bad actor giving it really long input. I don't think this cooldown is needed, but I'm not completely against it if someone else disagrees.

I think in addition to a cooldown a limit of how many times the command can be executing at once would also help.

Reply:
We're already limiting how many times the command can be executing at once with the `max_concurrency` decorator. That's my point, I don't think the cooldown adds anything on top of that.